extensions/mssql/src/bigDataClusterLookUp.ts

```diff
@@ -0,0 +1,140 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as sqlops from 'sqlops';
+import * as constants from './constants';
+import * as UUID from 'vscode-languageclient/lib/utils/uuid';
+
+export class SqlClusterLookUp {
+	public static async lookUpSqlClusterInfo(conn: sqlops.IConnectionProfile | sqlops.connection.Connection): Promise<ConnectionParam> {
```

Comment:
You don't need static functions, you can just export functions.

Reply:
This way provides the specific namespace `SqlClusterLookUp` in front of the public method. I will leave this for now.